tests/test_basic_ops_dict.py

```diff
@@ -185,13 +188,9 @@ def test_scientific_notation_float() -> None:
         ({"hello": {"a": 2}}, "", "missing"),
         ({"hello": {"a": 2}}, "hello", "missing"),
         ({"hello": "???"}, "", "hello"),
-        ({"hello": "${foo}", "foo": "???"}, "", "hello"),
```

Comment:
Can you explain the motivation for this behavior change?
I am kind of okay with it, but it will help if you can explain.

Reply:
`get(key, default_value=v)` is expected to either return the corresponding value if `key` exists in the config, or the default value `v` otherwise. In that case, `"hello" in cfg` is now True, so it would be very weird to return the default value when calling `cfg.get("hello", default_value=v)`.

Now, one may ask "Why is `"hello" in cfg` True?". The reason is that, if it was False, it would be weird for `OmegaConf.is_missing(cfg, "hello")` to be False.

Let me know if that makes sense :)